### PR TITLE
Remove links from spacetv.in

### PR DESCRIPTION
**Pull request by freearhey**
Expired links.

### CODE REVIEW

**Review by BellezaEmporium — APPROVED** [review]

File: channels/ru.m3u [hunk 1]
```diff
@@ -467,8 +467,6 @@ http://live-saha.cdnvideo.ru/saha/saha/playlist.m3u8
 https://live-vestinn.cdnvideo.ru/vestinn/nn24-khl/playlist.m3u8
 #EXTINF:-1 tvg-id="NikaTV.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="https://nikatv.ru/public/img/logo2.png" group-title="",Ника ТВ (576p) [Not 24/7]
 https://live-nikatv.cdnvideo.ru/nikatv/nikatv.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="NLOHD.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",НЛО HD (720p)
-http://spacetv.in/stream/DIPK8H19CP8/344.m3u8
```
Comment (Dum4G) [inline channels/ru.m3u:471]: https://xx001.vivat.live/t-0301.0f6f.d/sd/00047/2m/index.m3u8

Ukrainian channel

File: channels/ru.m3u [hunk 2]
```diff
@@ -689,12 +685,8 @@ https://strm.yandex.ru/kal/start/start0.m3u8
 https://sc.id-tv.kz/STV.m3u8
 #EXTINF:-1 tvg-id="StranaFM.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Music",Страна FM (720p) [Offline]
 http://live.stranafm.cdnvideo.ru/stranafm/stranafm_hd.sdp/playlist.m3u8
-#EXTINF:-1 tvg-id="StrashnoeHD.ru" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="",Страшное HD (720p)
-http://spacetv.in/stream/DIPK8H19CP8/226.m3u8
```
Comment (Dum4G) [inline channels/ru.m3u:693]: http://stream.backustv.ru:80/live/btv2/index.m3u8

**Review by Dum4G — CHANGES_REQUESTED** [review]

File: channels/ru.m3u (hunk 1 above)
Comment (freearhey) [reply channels/ru.m3u:471]: Added in #5500

File: channels/ru.m3u (hunk 2 above)
Comment (freearhey) [reply channels/ru.m3u:693]: Added in #5501

**Review by BellezaEmporium — APPROVED** [review]

**Review by Dum4G — APPROVED** [review]

**Pull request merged**